scripts/scil_bundle_voxel_label_map.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_verbose_arg)
+from scilpy.tractanalysis import compute_tract_counts_map
+from scilpy.tractometry.distance_to_centroid import min_dist_to_centroid
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Compute (upsampled) Nifti label image from bundle and '
+                    'centroid. Each voxel will have the label of its '
+                    'nearest centroid point',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file.')
+    p.add_argument('in_centroid',
+                   help='Centroid streamline corresponding to bundle.')
+    p.add_argument('reference',
+                   help='Nifti reference image.')
+    p.add_argument('output_map',
+                   help='Nifti image with corresponding labels.')
+    p.add_argument('--upsample',
+                   type=float, default=2,
+                   help='Upsample reference grid by this factor. '
+                        '[%(default)s]')
+
+    add_overwrite_arg(p)
+
+    add_verbose_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser,
+                        [args.in_bundle, args.in_centroid],
+                        optionnal=args.reference)
+    assert_outputs_exist(parser, args, args.output_map)
+
+    sft_bundle = load_tractogram_with_reference(parser, args,
```

Comment:
If we are not using an optional reference (--reference), this wrapper is useless. Simply using the load_streamlines and using the reference directly (since right now it is mandatory)